modules/gitops-enable-traffic-management-and-metric-plugins-in-argo-rollouts.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/configuring_traffic_management_and_metric_plugins_in_argo_rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enable-traffic-management-and-metric-plugins-in-argo-rollouts_{context}"]
+= Enabling Traffic management and Metric plugins in Argo Rollouts
+
+To enable traffic management and metric plugins in Argo Rollouts, complete the following steps.
+
+.Procedure 
+
+. Log in to the {OCP} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *{gitops-title}* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
++
+.Example for adding the traffic management and metric plugins configuration in the `RolloutManager` CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollouts
+  labels:
+    example: basic
```

Comment:
We can remove `.metadata.labels` and the `example: basic` label. They aren't needed for this procedure, and I don't think they are a value add, here.

Reply:
Fixed.